src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -23,7 +23,7 @@ Namespace Microsoft.CodeAnalysis.Semantics
                         ' It is not permissible to access the Left property of a BoundUserDefinedBinaryOperator unconditionally,
                         ' because that property can throw an exception if the operator expression is semantically invalid.
                         ' Fetching the left operand through IBinaryOperatorExpression is safe.
-                        If rightOperatorBinary.LeftOperand Is value.LeftOnTheRightOpt Then
+                        If rightOperatorBinary.LeftOperand Is Create(value.LeftOnTheRightOpt) Then
```

Comment:
@AlekseyTs  projection was in the original code, that  is why the projection was still there. the Create is only code added compared to the original code. what you ask will need some work, so can we do that in another PR so that we can make all test passed?

the original code, converted rightOpeartorBinary to IOperation so that it can use LeftOperand property which contains some logic over its original BoundNode. finding all those places and change all those back to BoundNode will require some work since original code was written in that way in the first place. basically we need to reverse the original code to the other way.